autorun.brs

```diff
@@ -23,6 +23,12 @@ Sub Main(args)
   gaa = GetGlobalAA()
   gaa.version = version
 
+  ' display cursor and hide it in a corner
+  gaa.touchScreen = CreateObject("roTouchScreen")
+  gaa.touchScreen.EnableCursor(true)
+  gaa.touchScreen.SetCursorPosition(0, 0)
```

Comment:
It works, but the cursor is always visible (except for fullscreen video playback), so maybe it makes sense to show it only when a mouse is attached?

![20230912_104636](https://github.com/OutOfAxis/fugo-player-bs/assets/1911270/42f5eb74-bb24-4698-942a-b851bbedb7ea)

```suggestion
  if gaa.touchScreen.IsMousePresent() then
      gaa.touchScreen.EnableCursor(true)
      gaa.touchScreen.SetCursorPosition(0, 0)
  endif
```